ledger/src/blocktree_processor.rs

```diff
@@ -117,7 +117,18 @@ fn process_entries_with_callback(
         if entry.is_tick() {
             // if its a tick, save it for later
             tick_hashes.push(entry.hash);
-            continue;
+            if bank.is_block_boundary(bank.tick_height() + tick_hashes.len() as u64) {
+                // if its a tick that will cause a new blockhash to be created,
```

Comment:
It's

Reply:
Oops yup, will also write a test after I get back from dinner